lambo

```diff
@@ -298,6 +309,30 @@ if [[ "$NODE" = true ]]; then
     fi
 fi
 
+# Create a database by passing in -c or --createdb with sqlite or mysql
+# Defining MYSQL_PWD to avoid insecure password warning... user beware
+if [[ "$CREATEDB" != "" ]]; then
+    if [[ "$CREATEDB" == "sqlite" ]]; then
+        echo "${green}Creating an SQLite database here: database/$PROJECTNAME.sqlite${reset}"
+        DB_CONNECTION="sqlite"
+        DB_NAME="database\/$PROJECTNAME.sqlite"
+        touch "database/$PROJECTNAME.sqlite"
+    elif [[ "$CREATEDB" == "mysql" ]]; then
+        echo "${green}Checking if a MySQL database named $PROJECTNAME already exists${reset}"
+        DBEXISTS=`MYSQL_PWD=$DB_PASSWORD mysql -u$DB_USERNAME -P$DB_PORT -h$DB_HOST -e "SHOW DATABASES" | grep $PROJECTNAME`
```

Comment:
Is there any other safe way to pass the command in without getting warnings?

Or could we possibly unset the env var afterward at least?

Reply:
Should this grep `$DB_NAME` instead of `$PROJECTNAME`?

Reply:
AFAIK this only sets `MYSQL_PWD` temporarily so it won't persist as it's not exported. At least, neither `printenv` nor `set` reveal it in lambo after this block.

Agreed, security is an issue here but I'm unaware of a better way that doesn't prompt for the password. We could prompt, though. Or use an API (e.g., PHP->PDO_MYSQL) and read the lambo configuration file. Setting a temporary file and using MySQL's `--defaults-extra-file=foo.cnf` is also an option but I don't feel this complexity is worth it. IMHO using `MYSQL_PWD` is okay here.

Whatever the case, I could write something up along the lines of "Using `--createdb` is a security risk for reasons X and Y" although I suspect this isn't a real-world lambo concern considering empty MySQL passwords are common in local setups.

Good catch, `$DB_NAME` was added but not fully implemented in the second commit, but is now.

Remaining question: for sqlite, many DB_ definitions (e.g., `DB_PASSWORD`) are set in `.env` but not used. Should we remove or set them to ""? Or leave as us to make switching easier later on?

Reply:
All makes sense! Let's just leave them for SQLite for later usage.

And yah... can you just add the warning and then we'll merge this? Thanks so much for your work on it--I'll use this flag often!

Reply:
Oh my, this PR is getting out of control :) 

While attempting to merge in the perl changes from the other PR, I notice this all introduces complexities. For example, if a password contains special characters then it'll break the expression... probably the same is true for sed. Or if the project name begins with a number then it's going to break the database name replacement due to my misguided use of back references.

This is your punishment for trusting me, and mine for thinking this simple desire would be easy to implement for the masses. I'll focus on getting this working the best I can and then lambo can go worry about edge cases after.